netlogo-gui/src/main/app/App.scala

```diff
@@ -847,6 +832,13 @@ class App extends
     showAboutWindow()
   }
 
+  /**
+   * This is called reflectively by the mac app wrapper.
+   */
+  def handleShowPreferences(): Unit = {
+    showPreferencesDialog()
```

Comment:
So let's say that for some reason the `HelpMenu` wants to pass an argument into `showAboutMenu`. The author of the change adds an argument to `showAboutMenu` and recompiles. He sees that `handleShowAbout` doesn't compile anymore. He looks at the method and realizes that he has to provide some sort of argument in order for his change to work on mac. In the hypothetical world where he doesn't have `handleShowAbout` to catch his mistake he makes the change and recompilation succeeds, he goes on his way, *never having noticed that he broke NetLogo on Mac*. you could certainly put comments on the other methods, but nothing gets people's attention like a broken compile.